Sources/Service.swift

```diff
@@ -82,4 +81,25 @@ open class Service {
             return .ok
         }
     }
+
+    open func handle<M: Parseable>(json: Any) -> Result<M> {
+
```

Comment:
Wat is hier uit fout? Ik denk iets met de generics maar vind dat niet terug in onze coding guidelines hoe we dat doen.